src/ensembl/src/content/app/species-selector/containers/species-search-panel/SpeciesSearchPanel.tsx

```diff
@@ -29,11 +36,32 @@ const SearchPanel = () => {
         <SpeciesSearchField />
         <SpeciesCommitButton />
       </div>
-      <div className={styles.assemblyWrapper}>
-        <AssemblySelector />
-      </div>
+      <SelectedAssembly />
     </section>
   );
 };
 
+const SelectedAssembly = () => {
+  const genomeId = useSelector(getCurrentSpeciesGenomeId);
```

Comment:
How about renaming this to `activeGenomeId` instead?

Reply:
I made more extensive changes to remove the code related to the assembly selector from the codebase, and now this line looks different.